samples/libffmpeg/compile.sh

```diff
@@ -1,6 +1,6 @@
 jextract -t libffmpeg \
   -I /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include \
-  -I /usr/local/Cellar/ffmpeg@4/4.4.2_3/include \
+  -I /usr/local/Cellar/ffmpeg@4/4.4.2_4/include \
```

Comment:
Are these changes deliberate?

Reply:
Yes. Recent home-brew update brought this specific version on my machine. I need this for testing samples.